src/api/networkApi.js

```diff
@@ -5,16 +5,23 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import requestClient from './axiosInstance';
+import { requestExplorerServiceV1 } from './axiosInstance';
 
 const networkApi = {
   getPeers() {
```

Comment:
I'd rename it to `getPeerList()` because `getPeers` might easily be miswritten as `getPeer` and vice versa.